exampleSite/config/_default/languages.yaml

```diff
@@ -1,6 +1,8 @@
 en:
   languageName: English
   weight: 1
+  ### copyright (markdown can be used) ###
+  copyright: "Copyright &copy; {{< current_year >}} Company Name. All Rights Reserved. [Dot Org Theme](https://github.com/cncf/dot-org-hugo-theme) for Hugo by [CNCF](https://www.cncf.io)."
```

Comment:
The term "All Rights Reserved" is now considered [obsolete](https://en.wikipedia.org/wiki/All_rights_reserved#Obsolescence). In light of this I think the copyright parameter can be defined in `/config/_default/hugo.yaml` as it should language agnostic.

Reply:
I would rather not constrain this parameter to be language-agnostic. Linux Foundation still uses "All Rights Reserved" but, of course, it can be removed if the use-case permits it.